packages/csharp/PortingAssistant/PortingAssistant.Telemetry/Models/MetricsBase.cs

```diff
@@ -19,20 +20,20 @@ public class MetricsBase
         /// This property uniquely identifies the customers using porting assistant 
         /// Auto populate this field for all the metric logs
         /// </summary>
-        private string _customerID;
-        public string CustomerID
+        private string _uniqueID;
+        public string UniqueID
         {
             get
             {
-                if (string.IsNullOrWhiteSpace(_customerID))
+                if (string.IsNullOrWhiteSpace(_uniqueID))
                 {
-                    _customerID = NetworkInterface.GetAllNetworkInterfaces()
+                     _uniqueID = Crypto.SHA256(NetworkInterface.GetAllNetworkInterfaces()
                                          .Where(nic => nic.OperationalStatus == OperationalStatus.Up && nic.NetworkInterfaceType != NetworkInterfaceType.Loopback)
                                          .Select(nic => nic.GetPhysicalAddress().ToString())
-                                         .FirstOrDefault();
+                                         .FirstOrDefault());
```

Comment:
Would this generate the same result on a machine with multiple network interfaces?